src/controllers/highlights/HighlightPhrase.hpp

```diff
@@ -80,6 +80,7 @@ class HighlightPhrase
      * Qt>=5.13.
      */
     static QColor FALLBACK_HIGHLIGHT_COLOR;
+    static QColor FALLBACK_SELF_MESSAGE_HIGHLIGHT_COLOR;
```

Comment:
I'm not super happy with this name, but can't come up with anything better. There should at least be a comment here explaining when the color is used